### PR TITLE
Docs: Add screen capture use case

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**
This pull request allows edits from maintainers. English is not my native language. Feel free to correct mistakes in the text, if there are any. :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,36 @@ something like this:
 </app-media-stream>
 ```
 
+You can use `app-media-stream` to record device screen.
```
Comment (cdata) [inline README.md]: `to record device screen => to record a device screen`

File: README.md [hunk 2]
```diff
@@ -109,6 +109,36 @@ something like this:
...
+
+To capture screen in Chrome use `{"mandatory": {"chromeMediaSource": "screen"}}`
```
Comment (cdata) [inline README.md]: `To capture screen => To capture the screen`

File: README.md (hunk 1 above)
Comment (FluorescentHallucinogen) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (FluorescentHallucinogen) [inline README.md]: Done.

**Comment by cdata** [comment]
@FluorescentHallucinogen thanks for the changes. Please also refer to my request in [this comment](https://github.com/PolymerElements/app-media/pull/8#pullrequestreview-34354386).

**Comment by FluorescentHallucinogen** [comment]
> Can you please also add a back-reference to the Paul Kinlan article about this (or a better reference, if you have one).

I don't mind to add it, but what's it for?

Looks like, unfortunately, as of today there is no single up to date source about extensionless (plugin-free) screen capturing in both, Chrome and Firefox browsers.

E.g. @PaulKinlan's https://paul.kinlan.me/screen-recording-with-getUserMedia-and-WebRTC/ article has no word about Firefox.

Seems, the https://www.webrtc-experiment.com/Pluginfree-Screen-Sharing/#why-screen-fails section of @muaz-khan's page and his explanations in the comments are most relevant reference.

**Comment by FluorescentHallucinogen** [comment]
@cdata Where exactly in the text should I add the reference? This pull request allows edits from maintainers. Could you please help me and add it yourself.

**Comment by cdata** [comment]
It will be sufficient to add a note to the bottom of the explanation linking to the articles you just mentioned.

I appreciate your feedback. I do not have time to modify your PR for you. Please address the feedback, or else wait patiently until we have time to fix it ourselves.

**Comment by FluorescentHallucinogen** [comment]
Done.

**Comment by cdata** [comment]
Thanks for the contribution!

**Pull request merged**